tests/test_rst.py

```diff
@@ -53,3 +53,33 @@ def test_rst_raw():
 """, stream=warnings) is None
 
     assert '"raw" directive disabled' in warnings.getvalue()
+
+
+def test_rst_empty_file():
+    warnings = io.StringIO()
+    assert render("", stream=warnings) is None
```

Comment:
@bhrutledge I added an example for an empty RST file which should now emit a warning.